apm-agent-plugins/apm-jdbc-plugin/src/test/java/co/elastic/apm/agent/jdbc/JdbcDbIT.java

```diff
@@ -49,6 +49,8 @@ public JdbcDbIT(String url, String expectedDbVendor) throws Exception {
             {"jdbc:tc:postgresql:10://hostname/databasename", "postgresql"},
             {"jdbc:tc:mariadb:10://hostname/databasename", "mariadb"},
             {"jdbc:tc:sqlserver:2017-CU12://hostname/databasename", "sqlserver"},
+            {"jdbc:tc:db2:11.5.0.0a://hostname/databasename", "db2"},
+//            {"jdbc:tc:oracle://hostname/databasename", "oracle"},
```

Comment:
what about this one? a leftover?

Reply:
Yes, I tried it, but it requires building an image (or using some unofficial image) so I decided to keep out. I will remove the comment.